OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/pointsto/AbstractPointsToAnalysis.scala

```diff
@@ -543,11 +632,50 @@ trait AbstractPointsToAnalysis extends PointsToAnalysisBase with ReachableMethod
 
                 Results(results)
 
+            case UBP(fai: MethodFieldReadAccessInformation) =>
+                val defSite = e match {
+                    case ds: DefinitionSite               => ds
+                    case (_: Context, ds: DefinitionSite) => ds
+                }
+                // IMPROVE: Only handle new accesses
```

Comment:
Can we do this?

Reply:
Theoretically, yes. However as this is in the `continuationForShared`, i am quite unsure how to best track "old" eoptp or rather seen accesses per PC independently of the other continuation.

Reply:
It requires some changes in the state to track already seen accesses on a per-PC basis. It should not be too hard to implement, just to get it right and understandable. Lets shortly discuss this next meeting.

Reply:
Did we reach a conclusion on this?